frontend/src/bundles/common/components/button/button.tsx

```diff
@@ -1,19 +1,57 @@
-import { Button } from '@mui/material';
+import {
+    Button,
+    type ButtonPropsColorOverrides,
+    type SxProps,
+    type Theme,
+} from '@mui/material';
 
 type Properties = {
     label: string;
     variant?: 'text' | 'outlined' | 'contained';
     type?: 'submit' | 'button';
+    disabled?: boolean;
+    className?: string;
+    sx?: SxProps<Theme>;
```

Comment:
className prop allows for modular css:
![image](https://github.com/BinaryStudioAcademy/bsa-2023-bsa-talents/assets/90120572/7945b016-e276-458c-9619-6855ba210fe4)


However, I'm having difficulty with the MUI specificity.
Example:
`
.test {
    // This will get over-written by MUI styles
}

.test.addSpecificity {
   // This will work, since specificity same as MUI.
}
`

Reply:
To clarify this, sx will not be used?

Reply:
we will use this - https://mui.com/material-ui/customization/how-to-customize/#overriding-styles-with-class-names

Reply:
ClassNames should work as expected now.